gui/src/components/s3/account-management.vue

```diff
@@ -865,29 +873,21 @@ export default class CortxAccountManagement extends Vue {
     this.accountsList = res && res.data ? res.data.s3_accounts : [];
     this.s3Url = res.data && res.data.s3_urls ? res.data.s3_urls : [];
     this.isS3UrlNone = this.s3Url.length === 0;
-
-    const cms_res = await Api.getAll(apiRegister.csm_user);
-    if (cms_res && cms_res.data && cms_res.data.users) {
-      this.userData = cms_res.data.users;
-    }
-
-    this.isLoggedInUserAdminOrManage();
     this.$store.dispatch("systemConfig/hideLoader");
   }
 
-  public isLoggedInUserAdminOrManage() {
-    const usersData = this.userData.find((element: any) => {
-      if (
-        element.username.localeCompare(this.loggedInUserName, undefined, {
-          sensitivity: "base"
-        }) === 0
-      ) {
-        return true;
+  public async checkPermissions() {    
+    const routerApp: any = this;
```

Comment:
This is not routerApp. It is instance of vue. You can call it vueInstance instead of routerApp.

Reply:
Updated const name as vueInstance